### PR TITLE
 Fix link to Design Principles in PTransform Style Guide 

### PR DESCRIPTION
**Pull request by ttanay**
The href to Design Principles was broken in the PTransform Style Guide in the documentation. 
The href was changed to point to the Contribution Guide.

As per discussion in mailing list: https://lists.apache.org/thread.html/ade6f19bda020e825698e28613b3f8024b4da862181ef22bd5a9e3d8@%3Cdev.beam.apache.org%3E

------------------------

Thank you for your contribution! Follow this checklist to help us incorporate your contribution quickly and easily:

 - [x] [**Choose reviewer(s)**](https://beam.apache.org/contribute/#make-your-change) and mention them in a comment (`R: @username`).
 - [x] Format the pull request title like `[BEAM-XXX] Fixes bug in ApproximateQuantiles`, where you replace `BEAM-XXX` with the appropriate JIRA issue, if applicable. This will automatically link the pull request to the issue.
 - [x] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

Post-Commit Tests Status (on master branch)
------------------------------------------------------------------------------------------------

Lang | SDK | Apex | Dataflow | Flink | Gearpump | Samza | Spark
--- | --- | --- | --- | --- | --- | --- | ---
Go | [Build Status](https://builds.apache.org/job/beam_PostCommit_Go/lastCompletedBuild/) | --- | --- | --- | --- | --- | ---
Java | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Apex/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Dataflow/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Flink/lastCompletedBuild/)<br>[Build Status](https://builds.apache.org/job/beam_PostCommit_Java_PVR_Flink_Batch/lastCompletedBuild/)<br>[Build Status](https://builds.apache.org/job/beam_PostCommit_Java_PVR_Flink_Streaming/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Gearpump/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Samza/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Spark/lastCompletedBuild/)
Python | [Build Status](https://builds.apache.org/job/beam_PostCommit_Python_Verify/lastCompletedBuild/)<br>[Build Status](https://builds.apache.org/job/beam_PostCommit_Python3_Verify/lastCompletedBuild/) | --- | [Build Status](https://builds.apache.org/job/beam_PostCommit_Py_VR_Dataflow/lastCompletedBuild/) <br> [Build Status](https://builds.apache.org/job/beam_PostCommit_Py_ValCont/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PreCommit_Python_PVR_Flink_Cron/lastCompletedBuild/) | --- | --- | ---

See [.test-infra/jenkins/README](https://github.com/apache/beam/blob/master/.test-infra/jenkins/README.md) for trigger phrase, status and link of all Jenkins jobs.

### CODE REVIEW

**Comment by ttanay** [comment]
Test Failure:
```
- ./generated-local-content/beam/release/2016/06/15/first-release.html
  *  External link https://issues.apache.org/jira/browse/BEAM/ failed: response code 0 means something's wrong.
             It's possible libcurl couldn't connect to the server or perhaps the request timed out.
             Sometimes, making too many requests at once also breaks things.
             Either way, the return message (if any) from the server is: Couldn't connect to server
```

This is not related to the change in this PR.

**Comment by ttanay** [comment]
R: @mxm

**Comment by mxm** [comment]
Run Website PreCommit

File: website/src/contribute/ptransform-style-guide.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ _A style guide for writers of new reusable PTransforms._
 ### Consistency
 Be consistent with prior art:
 
-* Adhere to [Beam design principles]({{ site.baseurl }}/contribute/design-principles/).
+* Adhere to [Beam design principles]({{ site.baseurl }}/contribute/).
```
Comment (mxm) [inline website/src/contribute/ptransform-style-guide.md]: Should we adjust the title, e.g. `Please read the contribution guide`?

Comment (ttanay) [reply website/src/contribute/ptransform-style-guide.md]: Sure, I'll change the title.

**Comment by mxm** [comment]
Run Website PreCommit

**Comment by mxm** [comment]
Link checks fail. I've manually verified that the new link works :)

**Comment by ttanay** [comment]
Thank you @mxm :metal:

**Pull request merged**